frontend/src/pages/settings/index.tsx

```diff
@@ -0,0 +1,12 @@
+import SettingsContent from "./settings-content";
+
+export default function SettingsPage() {
+  return (
+    <div className="grid grid-rows-10 p-7 gap-4 w-full">
```

Comment:
у нас эта верстка на всех страницах используется, можно в компонент вывести, что-то типо такого:
```suggestion
export const PageLayout = ({ title, actionButton, refreshButton, contentComponent }) => (
    <div className="grid grid-rows-10 p-7 gap-4 w-full">
        <div className="flex items-center justify-start font-[700] font-pop text-[28px] gap-3 text-[#3F434A]">
            <p>{title}</p>
            {actionButton}
            {refreshButton}
        </div>
        {contentComponent}
    </div>
);
```